_posts/randy/2022-07-24-how-video-takes-all-day.md

```diff
@@ -0,0 +1,189 @@
+---
+layout: post
+title: "How Video Takes All Day"
+author: randy
+permalink: /@randy/how-video-takes-all-day
+image: "assets/img/how-video-takes-all-day.png"
+description: As video content gets shorter in step with modern attention spans, entertainment value needs to go up.  How do you figure out the creative parts and pieces in short order? Fortunately, thousands of media assets are available online, shaving eons off production time.
+tags: tails randy analog-sampling
+---
+
+<video width="720" height="auto" controls style="max-width: 100%">
+   <source src="/assets/how-video-takes-all-day-720.mp4" type="video/mp4">
+</video>
+
```

Comment:
Possibly worth adding a short thing to direct people to Part 1 if they have not read yet.  I know it comes later, but it's weird to get halfway through and then realize there's a part 1.  With the title change to include "Part 2" I think both of these makes things a lot more clear.

Reply:
doner